packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```diff
@@ -15,11 +16,11 @@ function setExtraConsole {
     source (loop0)/etc/os-release
     set baseExtraConsole="console=ttyS0"
     # rpi
-    if regexp "arm64-rpi" $KAIROS_ARTIFACT; then
+    if test $KAIROS_MODEL == "rpi3" -o test $KAIROS_MODEL == "rpi4"; then
```

Comment:
now that we have the model on its own var, this is much easier than before as we can exact match!

Reply:
very cool